lib/util/cache/package/index.ts

```diff
@@ -10,29 +10,33 @@ function getGlobalKey(namespace: string, key: string): string {
   return `global%%${namespace}%%${key}`;
 }
 
-export function get<T = any>(namespace: string, key: string): Promise<T> {
+export async function get<T = any>(
+  namespace: string,
+  key: string
+): Promise<T | undefined> {
   if (!cacheProxy) {
     return undefined;
   }
   const globalKey = getGlobalKey(namespace, key);
   if (memCache.get(globalKey) === undefined) {
     memCache.set(globalKey, cacheProxy.get(namespace, key));
   }
-  return memCache.get(globalKey);
+  const result = await memCache.get(globalKey);
+  return result;
 }
 
-export function set(
+export async function set(
   namespace: string,
   key: string,
   value: unknown,
   minutes: number
 ): Promise<void> {
   if (!cacheProxy) {
-    return undefined;
+    return;
   }
   const globalKey = getGlobalKey(namespace, key);
   memCache.set(globalKey, value);
-  return cacheProxy.set(namespace, key, value, minutes);
+  await cacheProxy.set(namespace, key, value, minutes);
```

Comment:
Was this a coding mistake before now?

Reply:
Not exactly, it were more liberal `async/await` usage rules

Reply:
Seems so